addon/components/o-s-s/smart/number-input.stories.js

```diff
@@ -0,0 +1,159 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::Smart::NumberInput',
+  component: 'smart-number-input',
+  argTypes: {
+    value: {
+      description: '[OPTIONAL] The current value of the input',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'number' }
+    },
+    min: {
+      description: '[OPTIONAL] The minimum value the number can be',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'number' }
+    },
+    max: {
+      description: '[OPTIONAL] The maximum value the number can be',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'number' }
+    },
+    step: {
+      description: '[OPTIONAL] The increase & decrease value of each button press. Defaults to 1',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: '1' }
+      },
+      control: { type: 'number' }
+    },
+    loading: {
+      description: '[OPTIONAL] Shows loading state with animated placeholder',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: 'false' }
+      },
+      control: { type: 'boolean' }
+    },
+    placeholder: {
+      description: '[OPTIONAL] Placeholder text shown during loading state',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'text' }
+    },
```

Comment:
I'm confused, does this actually work ? I thought you mentioned it was not possible in the PR description